com.fribbledom.telephant.json

```diff
@@ -0,0 +1,148 @@
+                    "dest": "src/github.com/gorilla/websocket"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/sirupsen/logrus",
+                    "commit": "7ea96a3284ed9d25de5056804d60426cb75833c7",
+                    "dest": "src/github.com/sirupsen/logrus"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/BurntSushi/toml",
+                    "commit": "3012a1dbe2e4bd1391d42b32f0577cb7bbc7f005",
+                    "dest": "src/github.com/BurntSushi/toml"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/dustin/go-humanize",
+                    "commit": "afde56e7acacd811f6c94228c2c61af2b0e93158",
+                    "dest": "src/github.com/dustin/go-humanize"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/muesli/go-app-paths",
+                    "commit": "c184ea3ba2f517461749fca7a793d9609caff57d",
+                    "dest": "src/github.com/muesli/go-app-paths"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/muesli/telephant",
+                    "branch": "master",
+                    "dest": "src/github.com/muesli/telephant"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/mitchellh/go-homedir",
+                    "commit": "af06845cf3004701891bf4fdb884bfe4920b3727",
+                    "dest": "src/github.com/mitchellh/go-homedir"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/tomnomnom/linkheader",
+                    "commit": "02ca5825eb8097f10d9cc53da78481a85ad84e04",
+                    "dest": "src/github.com/tomnomnom/linkheader"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/gen2brain/beeep",
+                    "commit": "ff0f4a39397e723435fb480420cfaf2bf3d1f835",
+                    "dest": "src/github.com/gen2brain/beeep"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/godbus/dbus",
+                    "commit": "06fc4b473149e499166adbb9e31c7365a8ea146f",
+                    "dest": "src/github.com/godbus/dbus"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/davecgh/go-spew",
+                    "commit": "d8f796af33cc11cb798c1aaeb27a4ebc5099927d",
+                    "dest": "src/github.com/davecgh/go-spew"
+                },
+                {
+                    "type": "file",
+                    "path": "com.fribbledom.telephant.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html